### DISCUSSION TITLE
Connecting to Emby after setting up with Jellyfin

### DISCUSSION THREAD

**Discussion by NumberSyx** [seerr-team/seerr#323] [Q&A]
I initially set this up and connected it to my Jellyfin, the issue is that I've since switched to Emby and I can't figure out how to point it at my Emby server. 

I've added the environment variable to my compose file and now my settings window has a tab titled "Emby", but when I try to library scan or import users it doesn't work and I don't see an option to point it at Emby. Any suggestions?

Thanks!

**Comment by fallenbagel** [comment]
I'm not sure if this would work but just in case
Backup `settings.json` and the dB folder after stopping jellyseerrr then settings.json only. Then restart it and go to
`http://jellyseerr:5055/setup` page and try. I'm not sure if this would work but worth a try
